src/components/content-tab-assistant.tsx

```diff
@@ -98,67 +109,69 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 	}, [ isAuthenticated ] );
 
 	const disabled = isOffline || ! isAuthenticated;
+
+	const renderAuthenticatedView = () => (
+		<>
+			<div className="text-gray-500 mb-4">
+				Welcome to the Studio assistant. I can help manage your site, debug issues, and navigate
+				your way around the WordPress ecosystem.
```

Comment:
Do we need to translate this string or will it be replaced by a server message at some point?